internal/server/iigointernal/judiciary.go

```diff
@@ -193,6 +313,17 @@ func checkMonotonicityOfSanctionThresholds(clientSanctionMap map[roles.IIGOSanct
 	return true
 }
 
+// unpackSingleIslandTransgressions fetches the rule names of any broken rules as per EvaluationReturns
+func unpackSingleIslandTransgressions(evaluationReturn roles.EvaluationReturn) []string {
```

Comment:
no test?

Reply:
Please see judiciary_test.go lines 622 to 672

Reply:
ah -- missed it- sorry and thanks

Reply:
Due to test added as above, this had been moved (lines 706 - 756)